README.md

```diff
@@ -23,6 +23,11 @@ ParseXbrl.parse('./test/sampleXbrlDocuments/amazon_10k.xml').then(function(parse
 });
 ```
 
+###### Example CLI usage:
+
+```
+parse-xbrl inputfile.xbrl outputfile.json
```

Comment:
I couldn't get this to work when I pulled it down using one of the sample docs in the test folder, any additional steps? 
parse-xbrl /parse-xbrl/test/sampleXbrlDocuments/amazon_10k.xml outputfile.json
zsh: command not found: parse-xbrl